roles/artemis/templates/application-prod.yml.j2

```diff
@@ -77,6 +77,11 @@ artemis:
       user: {{ user_management.jira.user }}
       password: {{ user_management.jira.password }}
       admin-group-name: {{ user_management.jira.admin_group}}
+	password-reset:
+	  credential-provider: TUMOnline
+	  links:
+		en: 'https://campus.tum.de/tumonline/ee/ui/ca2/app/desktop/#/pl/ui/$ctx/co_loc_password_reset.main?$ctx=design=ca2;header=max;lang=en'
```

Comment:
Please add variables for both links. and the provider e.g 
```
{{ usermanagement-provider-display-name }}
{{ usermanagement-passwort-reset-link-en }}
{{ usermanagement-passwort-reset-link-de }}
```

Add these variables to `roles/artemis/defaults/meta.yml` and provide some comments what that is, and why it is needed.